README.md

```diff
@@ -124,6 +124,20 @@ Example Playbooks
         main_config_file: ops_agent.yaml
 ```
 
+
+Compatibility
+-------------
+
+The matrix below lists the versions of this Ansible role and the agent versions it supports.
+
+| Ansible Role Version | Compatible Ops Agent Version(s) | Compatible Logging Agent Version(s) | Compatible Monitoring Agent Version(s) |
+|----------------------|-------------------------------- | ----------------------------------- | -------------------------------------- |
+| **1.x.x**            | 1.x.x                           | 1.x.x                               | 6.x.x                                  |
```

Comment:
Ops agent can be bumped to 2.x.x once #74 is checked in.

Reply:
Yup!